core/modules/czar/CzarConfig.cc

```diff
@@ -38,6 +41,22 @@ namespace {
 
 LOG_LOGGER _log = LOG_GET("lsst.qserv.czar.CzarConfig");
 
+void QservLogger(struct timeval const& mtime,
```

Comment:
Yes, it is very annoying, we should try to avoid repeating code.`util` is actually not a perfect place for that code, the library that we build from that package already has too many dependencies which are not needed for your xrootd plugin. LEt's say we keep it as it is now and then think about it at next refactoring.